2022q4/freebsd-foundation.adoc

```diff
@@ -0,0 +1,184 @@
+- follow-up commits to Mark Johnston's work to add ZFS Support to makefs(8)
+  (e.g.,
+  link:https://cgit.freebsd.org/src/commit/?id=89585511cc052643a774f64f6450d18e7dd51d4a[work
+  to easily provide ZFS-based VM and cloud images] and
+  link:https://cgit.freebsd.org/src/commit/?id=72a1cb05cd230ce0d12a7180ae65ddbba2e0cb6d[automation
+  for better defaults] from Li-Wen Hsu)
+- a variety of work from Ed Maste, including
+  link:https://cgit.freebsd.org/src/commit/?id=38a52bd3b5cac3da6f7f6eef3dd050e6aa08ebb3[an
+  ssh update] and
+  link:https://cgit.freebsd.org/src/commit/?id=86edb11e7491e657e6c75ef6814867021665c377[a
+  switch to LLVM objdump].
+
+==== Continuous Integration and Quality Assurance
+
+The Foundation provides a full-time staff member and funds projects to improve
+continuous integration, automated testing, and overall quality assurance efforts
+for the FreeBSD project.  You can read about the latest activity for that work
+in a separate report entry.
+
+==== FreeBSD Advocacy and Education
+
+Much of our effort is dedicated to Project advocacy.  This may involve
+highlighting interesting FreeBSD work, producing literature and video tutorials,
+attending events, or giving presentations. The goal of the literature we produce
+is to teach people FreeBSD basics and help make their path to adoption or
+contribution easier. Other than attending and presenting at events, we encourage
+and help community members run their own FreeBSD events, give presentations, or
+staff FreeBSD tables.
+
+The FreeBSD Foundation sponsors many conferences, events, and summits around the
+globe. These events can be BSD-related, open source, or technology events geared
+towards underrepresented groups. We support the FreeBSD-focused events to help
+provide a venue for sharing knowledge, working together on projects, and
+facilitating collaboration between developers and commercial users. This all
+helps provide a healthy ecosystem. We support the non-FreeBSD events to promote
+and raise awareness of FreeBSD, to increase the use of FreeBSD in different
+applications, and to recruit more contributors to the Project. We are continuing
+to attend events both in person and virtual as well as planning the November
+Vendor Summit. In addition to attending and planning virtual events, we are
+continually working on new training initiatives and updating our selection of
+link:https://freebsdfoundation.org/freebsd-project/resources/[how-to guides] to
+facilitate getting more folks to try out FreeBSD.
+
+Check out some of the advocacy and education work we did last quarter:
+
+* Sponsored the OpenZFS Developer Summit, October 24-25, 2022 in San Francisco, CA
+
+* Sponsored All Things Open, October 30-November 2, 2022 in Raleigh, NC
+
+* Sponsored and helped organize the November 2022 FreeBSD Vendor Summit. Videos
+from the Summit are available link:https://youtube.com/playlist?list=PLugwS7L7NMXwVfBq5eDRky450jp7LTRJj[here].
```

Comment:
I'd move more of the second sentence into the link text for better accessibility for screenreaders using out-of-text link lists, to avoid having a link reading just "here". Maybe `link:https://youtube.com/playlist?list=PLugwS7L7NMXwVfBq5eDRky450jp7LTRJj[Videos
from the Summit are available here].` to avoid changing the overall text.